tests/reftests/deps-only.test

```diff
@@ -85,15 +85,15 @@ depends: "b"
 ### opam install a c --deps-only
 [ERROR] Package conflict!
   * Missing dependency:
-    - a
+    - deps-of-c -> b -> a
```

Comment:
Here, the message was good as `a` is requested in the cli.

Reply:
Not really. `opam install --deps-only a` is totally valid so the missing dependency is really via `c`